tests/aequilibrae/project/test_project.py

```diff
@@ -0,0 +1,20 @@
+from unittest import TestCase
+import tempfile
+import os
+from aequilibrae.project import Project
+
+temp_proj_name = os.path.join(tempfile.gettempdir(), "test_file.sqlite")
+
+
+class TestProject(TestCase):
+    def tearDown(self) -> None:
+        if os.path.isfile(temp_proj_name):
+            os.unlink(temp_proj_name)
+
+    def test_creation(self):
+        test_file = temp_proj_name
+        with self.assertRaises(FileNotFoundError):
```

Comment:
you can also use

```
self.assertRaises(FileNotFoundError, Project, test_file)

Reply:
applied here, but I would need to change this in many other places.  Will try to learn this pattern and use it in the future